dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -121,19 +121,44 @@ public final PreparedStatement createStatementWithBoundFixtureSymbols(
     }
 
     @Override
+    public final StatementExecution createStatementExecutionWithBoundFixtureSymbols(TestHost th, String commandText)
+            throws SQLException {
+        return createStatementExecution(createStatementWithBoundFixtureSymbols(th, commandText));
```

Comment:
Right, the `PreparedStatement` variants can be made protected or merged into the new versions. So far I removed them from the interface. On the other extreme - there is an option to keep the old versions into the interface, maybe marking them `@Deprecated`.